cmd/veeam-sos-api.go

```diff
@@ -118,14 +108,11 @@ func veeamSOSAPIGetObject(ctx context.Context, bucket, object string, rs *HTTPRa
 	switch object {
 	case systemXMLObject:
 		si := systemInfo{
-			ProtocolVersion: "1.0",
-			ModelName:       "MinIO " + ReleaseTag,
+			ProtocolVersion: "\"1.0\"",
+			ModelName:       "\"MinIO " + ReleaseTag + "\"",
 		}
 		si.ProtocolCapabilities.CapacityInfo = true
 
-		// Default recommended block size with MinIO
-		si.SystemRecommendations.KBBlockSize = 4096
```

Comment:
You shouldn't be removing this.. this recommendation from MinIO. 

Default of 1MiB is not a good idea in general.

Reply:
Yes it is a conscious decision this is a recommendation, users are free to choose what they like on their end.

Reply:
We are not going to change this decision for now.

Reply:
> Given that the setting sets the S3 object size, it simultaneously affects read from source, block, file, dedup, and object storage backup targets for a specific Veeam job i.e. it has wide-ranging consequences.

It affects dedup efficiency. That's it. The clients will have to read the same amount of data.

Reducing the amount of objects to list/write/update by close to a factor of 4x has a much bigger impact than a potential single-digit-percentage improvement in storage efficiency. Either way it is a default you can override at your own discretion if you are that confident you know better.

Reply:
OK, thanks for the info. I am aware the value can be overridden on the client's side, but it has to be done for every backup job so having a sane default is valuable. I understand your argument that 4096 is precisely this sane default.